src/kitconcept/glossary/tests/test_views.py

```diff
@@ -68,3 +68,9 @@ class GlossaryViewTestCase(BaseViewTestCase):
     def setUp(self):
         super(GlossaryViewTestCase, self).setUp()
         self.view = api.content.get_view(u"view", self.g1, self.request)
+
+    def test_view(self):
+        try:
```

Comment:
@rodfersou what are you doing here? A try/except in a test?

Reply:
assert that not raises an exception..

Reply:
@rodfersou this does not make any sense to me TBH. You are replacing an exception that is thrown with a test failure.

Reply:
@tisto learned something new, will fix, thanks.